src/main/java/school/lemon/changerequest/java/introduction/hw1/Task10.java

```diff
@@ -1,9 +1,19 @@
 package school.lemon.changerequest.java.introduction.hw1;
 
 
-public class Task10 {
+public class Task10{
 
     public static boolean containDigitTwo(int n) {
-        return false;
+
+        String n2 = Integer.toString(n);
```

Comment:
It's great that you've found such sollution, but try do it without converting to string.